docs/manual/manual_advanced_arming.rst

```diff
@@ -13,6 +13,10 @@ simplifying flight controller integration.
 
 The :ref:`arming_module_support` section below details which Vertiq modules currently support advanced arming.
 
+.. note::
+    If your module does not support the :ref:`redundant throttle feature <redundant_throttle_manual>`, please also read the :ref:`legacy_arming_support` section for additional constraints 
+    on the advanced arming feature. You can find if your module's firmware supports redundant throttle on its family page.

```

Comment:
@vertiq-jordan One last thing, this note is good and the legacy section is good, I may just sprinkle this note in a few more places. People rarely read our docs from the top to bottom in their entirety, so there's a good chance they might miss this up here. Dropping the note into a few other places where we removed stuff, like Always Armed, Armed State Transitions, Consecutive Arming Throttles, etc. may make sure people actually get that message